podcastparser.py

```diff
@@ -304,6 +304,44 @@ def start(self, handler, attrs):
         handler.get_episode_attr('chapters').append(chapter)
 
 
+class ItunesOwnerAttr(Target):
+    WANT_TEXT = True
+
+    def end(self, handler, text):
+        if not self.overwrite and handler.get_episode_attr(self.key):
+            return
+        handler.set_itunes_owner_attr(self.key, self.filter_func(text))
+
+
+class ItunesOwnerItem(Target):
+    def start(self, handler, attrs):
+        handler.add_itunes_owner()
+
+
+class PodcastAttrExplicit(Target):
+    WANT_TEXT = True
+    _VALUES_MAPPER = {
+        'yes': True,
+        'explicit': True,
+        'true': True,
+        'no': False,
+        'clean': False,
+        'false': False,
```

Comment:
The [spec](http://web.archive.org/web/20120323002653if_/http://www.apple.com/itunes/podcasts/specs.html#explicit) shows that "yes", "no" and "clean" are the only possible values?

Reply:
[This spec](https://help.apple.com/itc/podcasts_connect/#/itcb54353390) shows that only "true" and "false" values are valid. I think it'd be better to support all of them.